<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registrationForm</class>
 <widget class="QWidget" name="registrationForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color:rgb(254, 192, 5);</string>
   </property>
   <property name="text">
    <string>Registration Form</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="VIPCheck">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Register as VIP Driver</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>470</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="errlbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>371</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Spaceship Number:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Production Year:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Spaceship Model:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Spaceship Color:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>re-type your password:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="usernameTxt"/>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="passwordTxt"/>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="retypeTxt"/>
    </item>
    <item row="3" column="4">
     <widget class="QLineEdit" name="numberTxt">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QComboBox" name="models"/>
    </item>
    <item row="5" column="4">
     <widget class="QLineEdit" name="yearTxt">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLineEdit" name="colortxt">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>models</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>usernameTxt</zorder>
  <zorder>usernameTxt</zorder>
  <zorder>passwordTxt</zorder>
  <zorder>label_3</zorder>
  <zorder>retypeTxt</zorder>
  <zorder>label_4</zorder>
  <zorder>numberTxt</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>yearTxt</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>lineEdit_6</zorder>
  <zorder>checkBox</zorder>
  <zorder>VIPCheck</zorder>
  <zorder>pushButton</zorder>
  <zorder>errlbl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
